### PR TITLE
Add project ideas for Gammapy as part of Astropy

### PR DESCRIPTION
**Pull request by cdeil**
@joleroi @taldcroft @eteq - Could you please read over the project descriptions for Gammapy (which we said could be part of Astropy via private email) and give feedback?

Initially I wrote up these more chatty descriptions on the Gammapy wiki:
https://github.com/gammapy/gammapy/wiki/Gammapy-GSoC-2016-ideas
I'm inclined to remove that page, but we could also keep it (and maybe adjust it) if you think it's useful.

### CODE REVIEW

**Comment by taldcroft** [comment]
@cdeil - I don't think there is a particular requirement to fit the description within a single paragraph.  If you feel that the chattier version would help to inform and excite the right students then feel free to include more details.  Otherwise I'm happy to merge this as-is.

**Comment by cdeil** [comment]
@taldcroft - I'm happy to have this merged as-is and remove the Gammapy wiki page.

Actually my goal here was to make it sounds less exciting and more complicated than it actually is (e.g. I put "grunt work" in the project description), because we can mentor 2 students max, and I already got emails for many good students in the past week or two based on the Gammapy wiki page.

:-)

**Comment by taldcroft** [comment]
Great!

File: gsoc/gsoc2016/ideas_astropy.md [hunk 1]
```diff
@@ -74,3 +74,43 @@ necessary changes to make the ERFA python API public.  This includes:
...
+*Astronomy knowledge needed:* None.
+
+*Programming skills:* Scientific python (Numpy, Scipy, Astropy), Web development (Python backend, Javascript frontend)
+
+#### Description
+
+[Gammapy](https://gammapy.readthedocs.org/en/latest/) is a Python package for professional gamma-ray astronomers.
+We are looking for a web developer with good Python, HTML and Javascript skills that is interested in building web pages and apps to display and browse gamma-ray data and maybe even launch Gammapy analyses.
+There's a few different projects we'd like to see realised, depending on your interests and skills.
+One option is to build a much-improved version of [TeVCat](http://tevcat.uchicago.edu/), that includes more image and catalog data and interactivity (maps that pan & zoom, search field for source name) with the general public as well as professional gamma-ray astronomers as the target. This project would mostly be front-end development, plus Python scripts to prepare the images and catalogs in suitable formats.
```
Comment (eteq) [inline gsoc/gsoc2016/ideas_astropy.md]: Maybe add a sentence on what TeVCat is?

File: gsoc/gsoc2016/ideas_astropy.md [hunk 2]
```diff
@@ -74,3 +74,43 @@ necessary changes to make the ERFA python API public.  This includes:
...
+Experience with X-ray or gamma-ray data analysis (e.g. Fermi-LAT) is a plus, but not a requirement.
+
+*Method knowledge needed:* Some experience in data analysis (e.g. images, regions) and statistics (e.g. Poisson noise). 
+
+*Programming skills:* Python (including pytest and Sphinx) and scientific python (Numpy, Scipy, Astropy)
+
+#### Description
+
+[Gammapy](https://gammapy.readthedocs.org/en/latest/) is a Python package for professional gamma-ray astronomers.
+We are looking for someone that's interested to work on a few small tasks, each taking a few weeks of the GSoC total time.
```
Comment (eteq) [inline gsoc/gsoc2016/ideas_astropy.md]: Maybe phrase this a bit more specifically like "...work on a few distinct data analysis tasks, each taking ..."?  Makes it seem at least somewhat more exciting.

**Comment by eteq** [comment]
@cdeil - I know this is already merged, but perhaps you can quick sneak in those changes in a PR?  Or if, you like them, I can just change it directly in master.

**Pull request merged**